src/components/Nav/Nav.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from 'styled-components/macro';
```

Comment:
```suggestion
import styled from 'styled-components';
```
굳이 macro 경로에서 가져오지 않아도 무방할거 같아요!

Reply:
네!